packages/components/src/index.ts

```diff
@@ -1,4 +1,6 @@
+import { SqProject } from "@quri/squiggle-lang/";
 export { SquiggleChart } from "./components/SquiggleChart";
 export { SquiggleEditor } from "./components/SquiggleEditor";
 export { SquigglePlayground } from "./components/SquigglePlayground";
 export { SquiggleContainer } from "./components/SquiggleContainer";
+export const newProject = () => SqProject.create();
```

Comment:
Not sure about this. This was something that I was thinking about doing just because you can get away with not using @quri/squiggle-lang if you only have this single function. Otherwise you need @quri/squiggle-lang as a dependency for just SqProject.create(). I thought to include it here. Interested in comments.